### ISSUE TITLE
Opening of browser fails if url contains port information

### ISSUE THREAD

**Issue by simongregorebner** [caqtdm/caqtdm#45]
```
export CAQTDM_EXEC_LIST='Archiver;chromium-browser http://ui-data-api.psi.ch:7777/prepare?channel=hipa-archive/&P '
```

**Comment by HelgeBrands** [comment]
the browser is opening the problem is on the Linux side because the colon is used for the separation! Put everything  into a script. This should solve the problem.

**Comment by simongregorebner** [comment]
actually yes thats the workaround, and yes my description was not fully accurate - sorry about that (the browser is opening yes)
(actually this was a placeholder ticket for myself to fix it later ; ) )

However I don't believe the problem lies on the linux side a 
```
chromium-browser http://ui-data-api.psi.ch:7777/
```
on the console works fine, so the chopping must happen already in caqtdm somehow ...

**Comment by HelgeBrands** [comment]
yes it is happening inside caqtdm because environment variables are separated in Linux with the same character you are using for your port number the http:// is handled separately!
